lib/traject/macros/met_csv.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Macros
+  # Macros for extracting values from CSV rows
+  module MetCsv
+    # @param header_or_index [String] the field header or index to accumulate
+
+    def generate_creator
+      lambda do |row, accumulator, _context|
+        display_pattern = row['Artist Display Name'] unless row['Artist Display Name'].empty?
+        display_pattern = "#{display_pattern}, #{row['Artist Suffix']}" unless row['Artist Suffix'].empty?
```

Comment:
Maybe - this can most certainly be better. But we're discussing a better structure for dealing with this, as well as reviewing what the pattern should look like when some of the data is blank. 

I'm not sure that should block this PR, and work through changes in the near future as we establish the creator pattern.